hypertrace-gradle-docker-plugin/src/main/java/org/hypertrace/gradle/docker/DockerPlugin.java

```diff
@@ -109,17 +112,17 @@ private boolean isReleaseVersion(Project project) {
   }
 
   private String getBranchTag() {
-    // Use the value of CIRCLE_BRANCH environment variable if defined (that is, the branch name used
-    // to build in CI), otherwise for local builds use 'test'
-    return getEnvironmentVariable("CIRCLE_BRANCH")
+      // Use the value of CIRCLE_BRANCH/ GHA_BRANCH environment variable if defined (that is, the branch name used
+      // to build in CI), otherwise for local builds use 'test'
+    return getFirstAvailableEnvironmentVariable(Arrays.asList("CIRCLE_BRANCH", "GHA_BRANCH"))
```

Comment:
See my comment on https://github.com/hypertrace/attribute-service/pull/45#discussion_r546695122 - I think we should keep the plugin using standard variables. Can we move the normalization of the ref format into this code?

Reply:
updated